updater/doc.go

```diff
@@ -161,41 +173,73 @@ progress reports automatically: TryUpdate
 
 	// After initializing our Updater instance...
 
-	type updateHandler struct {}
+	import (
+		"context"
+		"fmt"
 
-	func (e updateHandler) FetchUpdate(ctx context.Context, info *UpdateInfo) error {
-		download, err := someFunctionThatDownloadsAFile(info.GetURL())
-		if err != nil {
-			// Oops something went wrong
-			return err
-		}
+		"github.com/kinvolk/nebraska/updater"
+	)
+
+	func getInstanceID() string {
+		// Your own implementation here...
```

Comment:
Let's keep the person used before:
```suggestion
		// The caller's own implementation here...
```

And below as well